src/NUnitTestAdapter/NUnit3TestExecutor.cs

```diff
@@ -46,6 +46,15 @@ public interface INUnit3TestExecutor
         IFrameworkHandle FrameworkHandle { get; }
     }
 
+    public enum RunType
+    {
+        Unknown,
+        CommandLineLegacy,
+        CommandLineCurrentVsTest,
```

Comment:
Should be `VSTest` to follow branding; also, `VS` is a two-letter acronym and this is typically capitalized in the .NET ecosystem: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

Reply:
The problem with VSTest is that the spelling checked goes kadabang.

Reply:
But added VS to the abbreviation list